devel/irrlichtmt/Portfile

```diff
@@ -4,18 +4,17 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cmake 1.1
 
-github.setup        minetest irrlicht 1.9.0mt4
+github.setup        minetest irrlicht 1.9.0mt8
 name                irrlichtmt
 categories          devel
-platforms           darwin
```

Comment:
Why would you need to remove this?

Reply:
@l2dy `platforms darwin` is now implicit to MacPorts, and is being removed from ports by default.